src/services/pact-verifier.service.ts

```diff
@@ -7,27 +6,29 @@ import { PactProviderOptions } from '../interfaces/pact-provider-module-options.
 @Injectable()
 export class PactVerifierService {
   public constructor(
-    @Inject(PactModuleProviders.ProviderOptions) private readonly options: PactProviderOptions,
-    @Inject(PactModuleProviders.PactVerifier) private readonly verifier: Verifier
+    @Inject(PactModuleProviders.PactVerifier) private readonly verifier: Verifier,
+    @Inject(PactModuleProviders.ProviderOptions) private readonly options: PactProviderOptions
```

Comment:
This looks like a breaking change too - worth mentioning in the documentation.

Reply:
I think my IDE shuffled the imports, I changed them back to the original way